doc/managing-the-glusterd2-service.md

```diff
@@ -0,0 +1,54 @@
+# Managing the glusterd Service
+
+After installing GlusterFS and Glusterd2, you must start `glusterd2`
+service. The glusterd2 service serves as the Gluster elastic volume
```

Comment:
From the second line, we are going to repeat stuff.  As per the new doc, we are going to give them an insight about glusterd before reaching here. Its needed in the beginning for them to get a better understanding. If its in the beginning, this will be a repetition. will use the same snippet there.

Reply:
Setup and install is out of scope for this page. This page is for post installation steps

Reply:
@harigowtham I think we should give a brief introduction on what is glusterd2 in the user guide and then provide a link to an extended explanation, else the doc will be too long and might be not very user-friendly to someone who already knows what glusterd2 is.

Reply:
yes. we have planned to give only a brief introduction. detailed introduction will be given as a link to the docs (https://docs.gluster.org/en/latest/) once the documetation for gd2 is available. The point i wanted to convey here is, adding the explanation about gd2 again and again will make this less user friendly because of repetition. so we can remove it here.